src/main/java/com/echobox/api/linkedin/client/DefaultWebRequestor.java

```diff
@@ -279,6 +280,10 @@ public Response executePost(String url, String parameters, String jsonBody,
       // Allow subclasses to customize the connection if they'd like to - set their own headers,
       // timeouts, etc.
       customizeConnection(request);
+      
+      HttpContent content = request.getContent();
+      long length = (content != null) ? content.getLength() : 0;
+      httpHeaders.setContentLength(length);
```

Comment:
If you look at `HttpRequest` from `DefaultWebRequestor.getResponse (line 369)` the content length is already set by the library if there is content. Please set the content length header or force an empty JSON body here